credentials/alts/internal/authinfo/authinfo_test.go

```diff
@@ -44,6 +44,8 @@ const (
 )
 
 func (s) TestALTSAuthInfo(t *testing.T) {
+	testPeerAttributes := make(map[string]string)
+	testPeerAttributes["peer"] = "attributes"
```

Comment:
nit: make "peer" and "attributes" local strings, in case we have more places to refer them in the future.

Reply:
Fixed! Thanks for catching that.